### DISCUSSION TITLE
Session to ingest aggregated data

### DISCUSSION THREAD

**Discussion by bjamir** [Sunbird-cQube/community#838] [General]
@RNarmatha Could you kindly conduct a live session to ingest latest aggregated data for all the standard programs? we would also like to discuss the detailed processes involved in ingestion over existing data. we are available on 22-23 Nov between 10:00-4:00 PM

**Comment by RNarmatha** [comment]
Hi @bjamir, In cQube aggregated data can't be ingested directly. The proccess of ingestion in cQube - Ingesting the raw data through API, and get visualized. 
@bommalla- Please add your inputs here

**Reply by RNarmatha** [reply]
Please Share the Data

**Comment by RNarmatha** [comment]
[Nagaland_latest_data.zip](https://github.com/Sunbird-cQube/community/files/13424454/Nagaland_latest_data.zip)
Kindly follow this for  preparing the data files
